### ISSUE TITLE
TUX IOTBZH.svg : bad logo

### ISSUE THREAD

**Issue by kooltux** [PoggoAgency/Tux-Collection#6]
IoT.bzh logo is not the right one on TUX IOTBZH.svg (and png).

The official logo uses Sans Serif fonts. Attached, a fix using inkscape but I don't commit because it should be normalized through PS.

[TUX_IOTBZH_fixed.svg.zip](https://github.com/PoggoAgency/Tux-Collection/files/3876017/TUX_IOTBZH_fixed.svg.zip)

**Comment by PoggoAgency** [comment]
<img width="595" alt="Capture d’écran 2019-11-22 à 12 42 42" src="https://user-images.githubusercontent.com/46819174/69427071-03681c00-0d2f-11ea-83a8-a5e8c07b72d1.png">

<img width="598" alt="Capture d’écran 2019-11-22 à 12 42 31" src="https://user-images.githubusercontent.com/46819174/69427098-1549bf00-0d2f-11ea-9327-98fed33b83a0.png">

Do we drop these two ones? In the very beginning, we told Fulup that we should do a graphic charter. He answered this logo was meant to live through different styles, as iot.bzh is not a common company!

**Comment by kooltux** [comment]
I would agree with you on the charter: I would'n drop the two ones, but simply update the logo on the first one.

**Comment by kooltux** [comment]
I would agree with you on the charter: I wouldnt drop those two ones, but simply update the logo on the first one. (thats what I did in the svg I posted earlier)
